tests/test_browser.py

```diff
@@ -3151,6 +3110,7 @@ def test_mainloop_infloop(self):
   def test_emterpreter_async_iostream(self):
     self.btest('emterpreter_async_iostream.cpp', '1', args=['-s', 'EMTERPRETIFY=1', '-s', 'EMTERPRETIFY_ASYNC=1'])
 
+  @no_chrome("required synchronous wasm compilation")
```

Comment:
Shall we make a separate decorator for this so we can easily find all the tests that use sync compilation?  `@needs_synchronous_compilation`?

Reply:
Good idea, adding.